project-generated-code-backend/Backend/Model/Accounts/Account.cs

```diff
@@ -18,6 +18,7 @@ public abstract class Account : Entity
         protected String contact;
         protected String email;
         protected Address address;
+        protected String password;
 
         public string Name { get => name; }
         public string Surname { get => surname; }
```

Comment:
This document seems that it will be shared between others, so the previous comment doesn't apply on situations like this one.